frontend/src/component/user/Profile/PasswordTab/PasswordTab.tsx

```diff
@@ -49,53 +58,29 @@ const EditProfile = ({
             setError(undefined);
             try {
                 const path = formatApiPath('api/admin/user/change-password');
-                const res = await fetch(path, {
+                await fetch(path, {
                     headers,
                     body: JSON.stringify({ password, confirmPassword }),
                     method: 'POST',
                     credentials: 'include',
                 });
```

Comment:
This could be moved into an API hook

Reply:
Addressed in https://github.com/Unleash/unleash/pull/2109/commits/4160084052638f5722c8ef8c28dbb0c753cfd89b